torch/ao/quantization/pt2e/numeric_debugger.py

```diff
@@ -11,3 +20,188 @@ def generate_numeric_debug_handle(graph_module: GraphModule) -> None:
         if node.op != "placeholder" and NUMERIC_DEBUG_HANDLE_KEY not in node.meta:
             node.meta[NUMERIC_DEBUG_HANDLE_KEY] = unique_id
             unique_id += 1
+
+
+class OutputLogger(torch.nn.Module):
```

Comment:
Does this need a `backwards_compatibility` annotation? Is it a public torch API?

Reply:
this is a public API, I'll add it to `torch.ao.quantization`, but we have not been using `backwards_compatibility` decorators.